### PR TITLE
any and custom message added. @todo 1

### PR DESCRIPTION
**Pull request by harikt**

### CODE REVIEW

**Comment by harikt** [comment]
@pmjones This has a test for the all and it passes :+1:  ;)

File: src/Aura/Filter/RuleCollection.php [hunk 1]
```diff
@@ -276,6 +276,16 @@ public function getTranslator()
      */
     protected function addRule($field, $method, $name, $params, $type)
     {
+        if ($name == 'all') {
+            $list = array_shift($params);
+            foreach ($list as $args) {
```
Comment (harikt) [inline src/Aura/Filter/RuleCollection.php:281]: May be we should test here , it is an array and throw an exception ?

**Comment by pmjones** [comment]
The only problem with this is that the "All" rule should be a separate class, just like the "Any" rule is, instead of being a behavior inside the RuleCollection.

**Comment by pmjones** [comment]
For what it's worth, everything aside from the "all" rule looks OK.  Maybe submit a separate pull request without the "all" rule stuff.

**Comment by harikt** [comment]
@pmjones I don't quite understand why you think the `all` should be separate. First I started with it, but later I understood it cannot give all the error messages as an array. But just mention all the subrules are not valid. May be we can rectify that also. But I would love to hear why you think so.

**Comment by harikt** [comment]
closing with PR #22

**Pull request closed without merging**